embedded-workshop-book/src/data-stage.md

```diff
@@ -25,14 +30,17 @@ To implement responding to a GET_DESCRIPTOR Device request, extend `usb-3.rs` so
 - `bNumConfigurations = 1`, must be at least `1` so this is the minimum value
 
 >(\*) the `common` crate refers to the crate in the `advanced/common` folder. It is already part of the `firmware` crate dependencies.
+**Use the `usb2::device::Descriptor` abstraction**
```

Comment:
I think you want a newline between this line and the preceding line that starts with `> `. As it is right now the "use (..)" text is part of the blockquote.